protocol/contracts/v2/strategies/RamosStrategy.sol

```diff
@@ -0,0 +1,186 @@
+     * This will be used to report equity performance: `sum(asset value in STABLE) - debt`
+     * The conversion of each asset price into the stable token (eg DAI) will be done off-chain
+     *
+     * @dev The asset value may be stale at any point in time, depending onthe strategy. 
+     * It may optionally implement `checkpointAssetBalances()` in order to update those balances.
+     */
+    function latestAssetBalances() public override view returns (AssetBalance[] memory assetBalances, uint256 debt) {
+        // RAMOS strategy assets = RAMOS's DAI balance + claimed AURA & BPT rewards =
+        // (bpt.balanceOf(RAMOS) / bpt.totalSupply() * Total_DAI_Balance in the LP) + 
+        // claimed AURA & BAL rewards
+
+        // get the RAMOS's DAI balance
+        AMO__IAuraStaking amoStaking = AMO__IAuraStaking(ramos.amoStaking());
+        AMO__IPoolHelper poolHelper = AMO__IPoolHelper(ramos.poolHelper());
+        IERC20 bptToken = IERC20(ramos.bptToken());
+        address stable = ramos.stable();
+
+        uint256 stableBalanceInRamos;
+        {
+            // Use `bpt.getActualSupply()` instead of `bpt.totalSupply()`
+            // https://docs.balancer.fi/reference/lp-tokens/underlying.html#overview
+            // https://docs.balancer.fi/concepts/advanced/valuing-bpt.html#on-chain
+            uint256 bptTotalSupply = IBPT(address(bptToken)).getActualSupply();
+            if (bptTotalSupply > 0) {
+                uint256 bptBalanceInRamos = amoStaking.stakedBalance() + bptToken.balanceOf(address(amoStaking));
+                (, uint256 stableBalanceInLP) = poolHelper.getTempleStableBalances();
+
+                stableBalanceInRamos = stableBalanceInLP * bptBalanceInRamos / bptTotalSupply;
+            }
+        }
+
+        // get the RAMOS strategy assets
+        uint256 length = assets.length;
+        assetBalances = new AssetBalance[](length);
+
+        address asset;
+        uint256 ramosStrategyBalance;
+        for (uint256 i; i < length; ++i) {
+            asset = assets[i];
+
+            if (asset == stable) {
+                // Sum the RAMOS balance and this contract's balance of the ERC20.
+                ramosStrategyBalance = stableBalanceInRamos + IERC20(asset).balanceOf(address(this));
+            } else {
+                // Sum the `amoStaking` balance, `rewardsRecipient` balance and this contract's balance of the ERC20.
+                // But since `rewardsRecipient` address may be a hot wallet with a bunch of different things going on, we won't include that balance
+                ramosStrategyBalance = IERC20(asset).balanceOf(address(amoStaking)) + IERC20(asset).balanceOf(address(this));
+            }
+
+            assetBalances[i] = AssetBalance({
+                asset: asset,
+                balance: addManualAssetBalanceDelta(ramosStrategyBalance, asset)
+            });
+        }
+
+        debt = currentDebt();
+    }
+
```

Comment:
To make the UX easier, should we add pass-through functions to the Ramos quote calls? Then the user/FE/whatever only needs to interact with the strategy contract only